.rubocop_base.yml

```diff
@@ -5,6 +5,8 @@ Layout/AlignHash:
   EnforcedColonStyle: table
 Layout/ExtraSpacing:
   AutoCorrect: true
+Layout/PercentLiteralDelimiters:
```

Comment:
Isn't this a `Style` cop? https://github.com/bbatsov/rubocop/blob/v0.52.1/lib/rubocop/cop/style/percent_literal_delimiters.rb#L13-L16

Also, wouldn't we want to set a preferred delimiter rather than disabling the cop completely?
cc @Fryguy

Reply:
According to http://rubocop.readthedocs.io/en/latest/cops_layout/#layoutspaceinsidepercentliteraldelimiters no :confused:

Reply:
That's a different cop.

Reply:
OH :boom: sorry then :sob: